.github/workflows/build.yml

```diff
@@ -0,0 +1,25 @@
+on:
+  schedule:
+  - cron: '00 00 * * MON'
+  workflow_dispatch:
+
```

Comment:
Perhaps at least try to run it once in this PR and see how it goes?
If yes, adding these:
```
  pull_request:
    branches: [main]
```

Also, before merging this PR into the main branch here, for CI experiment, probably you can merge this PR in your forked repo first and see whether the automatic PR works as expected.

Reply:
Sounds good, I'll try this out and let you know what the results look like.